internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -823,6 +827,7 @@ components:
         - gcp
         - gcp-rhui
         - guest-image
+        - guest-image-bootc
```

Comment:
Is it feasible to just keep the image types the same name, and put something in the image-request? Like as soon as you specific `bootc: {something...}` in the IR with the image type being `guest-image`, composer will go ahead and make an bootc'd image for you for the requested type. ITs that aren't supported would just 400 ofc.